### DISCUSSION TITLE
Setting the keyboard layout

### DISCUSSION THREAD

**Discussion by Fuhrerbaumanki** [prasanthrangan/hyprdots#305] [Q&A] [answered]
Please tell me how to change/add a second keyboard layout. There is not a lot of information on hyprland on the Internet in general, but I managed to find the configuration I needed. Here she is:
```
input {
    kb_layout = us,pl
    kb_options = grp:alt_shift_toggle
}
```
But I can’t figure out which configuration file to insert it into.

**Comment by T-Crypt** [comment, marked as answer]
Check out #306

**Comment by Fuhrerbaumanki** [comment]
I added only 2 lines to the configuration file Kb layout=us,pl. Kb
Parameters grp:alt_shift_toggle. Now I can print the keyboard layout. The
idea with the script is good, but for now I will not add it to my system.

вт, 10 окт. 2023 г., 16:16 Trevin ***@***.***>:
